### PR TITLE
version 1.3

### PR DESCRIPTION
**Pull request by AlexAndrei11**
Inca nu e gata, am vrut doar sa intreb daca e cv gresit pana acum sa corectez din timp

### CODE REVIEW

File: product/Product.h [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#include <iostream>
```
Comment (CristianIonutIonescu) [inline product/Product.h:1]: you already include iostream in your .cpp file, not needed here

File: product/Product.h [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+#include <string.h>
+
+class Product{
+    private:
```
Comment (CristianIonutIonescu) [inline product/Product.h:5]: https://google.github.io/styleguide/cppguide.html#Declaration_Order

File: store/Store.h [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#include <iostream>
```
Comment (CristianIonutIonescu) [inline store/Store.h:1]: same comments asa for product.h

File: product/Product.cpp [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+Product::Product(): name("-"), price(0), quantity(0) {}
+
+Product::Product(std::string name, float price, int quantity): name(name), price(price), quantity(quantity) {
+    std::cout << "Product created: Name = " << name << ", Price = " << price << ", Quantity = " << quantity << std::endl; 
+} 
+
+Product::Product(const Product &p): name(p.name), price(p.price), quantity(p.quantity) {}  // Copy Constructor
+
+Product::Product(Product&& p) = default; // Move Constructor
```
Comment (CristianIonutIonescu) [inline product/Product.cpp:13]: even though this is correct, homework is about all constructors, please implement them

**Review by CristianIonutIonescu — CHANGES_REQUESTED** [review]
Overall good. Please take care of the comments I've added

File: product/Product.cpp (hunk 4 above)
Comment (AlexAndrei11) [reply product/Product.cpp:13]: Aici oare te refereai la Move Constructor si la Move Assignment Operator ? Caci acelea le-am lasat cu default.

Comment (CristianIonutIonescu) [reply product/Product.cpp:13]: Da, cele 2

File: product/Product.h (hunk 1 above)
Comment (CristianIonutIonescu) [reply product/Product.h:1]: still not resolved

**Review by CristianIonutIonescu — APPROVED** [review]

File: product/Product.h (hunk 1 above)
Comment (AlexAndrei11) [reply product/Product.h:1]: Am scos #include <iostream> din Product.cpp caci nu ma lasa sa imi scriu fisierul .h fara el. Aveam nevoie pentru string-uri

Comment (CristianIonutIonescu) [reply product/Product.h:1]: trebuia pastrat in Product.cpp si sters de aici.
De asemenea, trebuie sa incluzi <iostream> in orice fisier sursa ai nevoie de a scrie/citi ceva. Nu trebuie pus intr-un singur loc.

**Pull request closed without merging**